models/convert/paths_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package convert
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/onflow/flow-go/ledger"
+)
+
+func TestPathsToBytes(t *testing.T) {
+	path1b := []byte("aac513eb1a0457700ac3fa8d292513e1")
+	path2b := []byte("1454ae2420513f79f6a5e8396d033369")
+	path3b := []byte("e91a3eb997752b78bab0bc31e30b1e30")
+
+	path1, _ := ledger.ToPath(path1b)
+	path2, _ := ledger.ToPath(path2b)
+	path3, _ := ledger.ToPath(path3b)
+
+	paths := []ledger.Path{path1, path2, path3}
+
+	got := PathsToBytes(paths)
+
+	assert.Equal(t, [][]byte{path1b, path2b, path3b}, got)
+}
+
+func TestBytesToPaths(t *testing.T) {
+	t.Run("nominal case", func(t *testing.T) {
+		path1b := []byte("aac513eb1a0457700ac3fa8d292513e1")
+		path2b := []byte("1454ae2420513f79f6a5e8396d033369")
+		path3b := []byte("e91a3eb997752b78bab0bc31e30b1e30")
+
+		path1, _ := ledger.ToPath(path1b)
+		path2, _ := ledger.ToPath(path2b)
+		path3, _ := ledger.ToPath(path3b)
```

Comment:
Same comment, this is dangerous. Not only does it assume it won't error (it might, there have been changes before), but it also relies on external logic to test our own logic, which we should not do.